### ISSUE TITLE
Aliases and human IDs - TODOs remaining ater initial release of experminental aliases PR

### ISSUE THREAD

**Issue by jldec** [opral/monorepo#2346]
Capture TODOs after #2108

- [ ] review docs
- [ ] handle (or disallow) alias non-uniqueness 
- [ ] handle (or disallow) alias updates
- [ ] review Fink search by alias https://github.com/opral/monorepo/pull/2108/files#r1510753012

**Comment by jldec** [comment]
Closing in favor of https://github.com/opral/inlang-message-sdk/issues/23 (MESDK-59)
